recipes-kernel/linux/linux-mainline/arm/defconfig

```diff
@@ -2280,19 +2942,33 @@ CONFIG_DRM_MIPI_DSI=y
 # CONFIG_DRM_DEBUG_SELFTEST is not set
 CONFIG_DRM_KMS_HELPER=m
 CONFIG_DRM_KMS_FB_HELPER=y
+# CONFIG_DRM_DEBUG_DP_MST_TOPOLOGY_REFS is not set
 CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=300
 # CONFIG_DRM_FBDEV_LEAK_PHYS_SMEM is not set
 # CONFIG_DRM_LOAD_EDID_FIRMWARE is not set
 # CONFIG_DRM_DP_CEC is not set
 CONFIG_DRM_GEM_CMA_HELPER=y
 CONFIG_DRM_KMS_CMA_HELPER=y
+CONFIG_DRM_GEM_SHMEM_HELPER=y
 CONFIG_DRM_SCHED=m
-CONFIG_DRM_LIMA=m
```

Comment:
LIMA DRM disabled ??? Does the LIMA driver still work ?

Reply:
Yes, it does. As well without the lima.conf as with the short defconfig from sunxi layer xcsoar crashes...

Reply:
Sorry, i don't get your point ...
At which condition XCSoar is crashing ?

Reply:
XCSoar could not start...
But otherwise it works well

Reply:
it could not start, but works well ??

Reply:
Sorry, without this defconfig and/or lima.conf it couldn't start
With both files it works well...